src/app/toolbar/toolbar.component.scss

```diff
@@ -19,6 +19,33 @@
   background: #eaefff;
   color: #000000;
 }
+mat-toolbar-row {
+  padding: 0;
+}
+
+.logo-toolbar-wrapper,
+.signed-in-toolbar-wrapper,
+.builderReport-toolbar-wrapper {
+  mat-toolbar-row {
+    padding: 0 20px;
+  }
+
+  @media all and (min-width: 768px) {
+    mat-toolbar-row {
+      max-width: 728px;
+      margin-left: auto;
+      margin-right: auto;
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  @media all and (min-width: 1152px) {
+    mat-toolbar-row {
+      max-width: 1024px;
+    }
+  }
```

Comment:
Where are the styles for `@media all and (min-width: 1152px)`?

Reply:
Same as the other one, we only need to update the max width

Reply:
Nah, I meant for the `1440px`. Check the other pages.